ee/tabby-db/migrations/0025_user-events.up.sql

```diff
@@ -1,11 +1,9 @@
 CREATE TABLE user_events (
     id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
-    completion_id TEXT NOT NULL,
-    user_id INTEGER NOT NULL,
+    user_name TEXT NOT NULL,
```

Comment:
```suggestion
    user_id INTEGER NOT NULL,
```

the `user` string in header is simply `ID`.